_build/data/transport.core.menus.php

```diff
@@ -227,7 +227,7 @@
 $children[1]->addMany($childrenOfClearCache, 'Children');
 
 /* Remove Locks */
-$children[2]= $xpdo->newObject('modMenu');
+/*$children[2]= $xpdo->newObject('modMenu');
```

Comment:
You can just remove code that's no longer needed instead of commenting it out; it'll still be in the git history if needed to revert at some point.